tools/RAiDER/cli/raider.py

```diff
@@ -273,3 +229,32 @@ def drop_nans(d):
                 if d[key][k] is None:
                     del d[key][k]
     return d
+
+
+##########################################################################
+def main(iargs=None):
+    # parse
+    inps = parseCMD(iargs)
+
+    # Read the template file
+    params = read_template_file(inps.customTemplateFile)
+
+    # Argument checking
+    params = checkArgs(params)
+
+    params['download_only'] = inps.download_only
```

Comment:
This should already be handled by the `read_template_file` function, with the default of False